src/formats/bbcode.js

```diff
@@ -539,23 +537,20 @@
 			allowsEmpty: true,
 			tags: {
 				img: {
-					src: null
+					src: null,
+					[EMOTICON_DATA_ATTR]: false
```

Comment:
Which browsers support this variable key notation?

Reply:
[Is this what you're looking for?](https://caniuse.com/mdn-javascript_operators_object_initializer_computed_property_names)

Reply:
Yes it is.